### ISSUE TITLE
Navigating away message when you haven't edited anything

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-framework#5514]
When navigating up and down the site tree you get messages to say you changes will be lost when nothing has been edited. It does the same in Chrome and IE10 so assume this is all browsers, although it doesn't happen every time you change page so I'm unsure of what is triggering the notification.
![pasted_image_11_05_16__4_21_pm](https://cloud.githubusercontent.com/assets/555033/15169951/2cabdaae-1795-11e6-9ab5-6df81242fdc2.png)

**Comment by chillu** [comment]
I can't reproduce this on latest `master` in Chrome on a standard `Page` type. It does happen with https://github.com/silverstripe-labs/silverstripe-frameworktest installed on the "BasicFields" test page, so some field on that page triggers it (maybe UploadField?)

**Comment by clarkepaul** [comment]
I'll see if I can reproduce today.

**Comment by clarkepaul** [comment]
Somehow I've got a page in a state which wasn't recognised as being saved–both save and publish buttons show as being not being saved even if no changes have been made to the page).

So by navigating to the page for the first time it looks as per below, even though no changes have been made. After saving the page it hasn't happen since.
![pasted_image_12_05_16__1_10_pm](https://cloud.githubusercontent.com/assets/555033/15201249/f621f14a-1842-11e6-9670-eefc398e4fd7.png)

Just talked to Ingo and he worked out it was TinyMCE sanitising the HTML without any visual changes eg adding <p> tabs around text which hasn't got any. In normal circumstances this wouldn't occur as the page would be sanitised upon save. 

Closing issue
